docs/DeveloperGuide.md

```diff
@@ -1,25 +1,88 @@
 # Developer Guide
 
-## Design & implementation
+## Design 
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Project overview
+
+NUSFOODREVIEW is built using java. It has cross platform ability that allows it to run on Windows, MAC-OS and Linux. 
+When the application is running it has to ability to display a number of canteens around NUS and all their stores available.
+The application allow users to read/enter reviews and ratings on the chosen store. User is also able to view some sample 
+menus of the stores. The application will record the timestamp whenever a new review is entered and print it out together 
+with the review when requested. Admin must verify themselves using password given by the application developer and change 
+it once they logged in as admin. Admin has the ability to add/delete canteen, store and review. NUSFOODREVIEW has a storage file
+that allows the saving and loading of data. This allows all parameters of the data to be stored and read whenever is needed.
+
+##Implementation
+Below is the list of commands available.
+
+* Display selected store sample menu: `menu`
+* Display all reviews of the selected store : `reviews`
+* Add a new review of the selected store: `add`
+* Goes back to home page to select canteen: `home`
+* Display all the stores of the selected canteen: `list`
+* Exiting the application: `exit`
+
+#### Sequence Diagram for `admin`
+When the user enters `2` to go into the admin page, the AdminVerification() is called. It will ask the user to input 
+the password. Then it will check the input against the set password. If fails then the user have to enter again or enter
+`exit` to exit the application.
+
+###[Proposed] delete feature 
```

Comment:
Hi there! It seems that your text may be incorrectly formatted. It could be due to the missing space after ###.

![image](https://user-images.githubusercontent.com/77775515/113081651-aaa27080-920b-11eb-8583-95c4d75e5036.png)